src/color.js

```diff
@@ -1,179 +1,201 @@
-    'mediumvioletred': [ 199 / 255, 21 / 255, 133 / 255 ],
-    'midnightblue': [ 25 / 255, 25 / 255, 112 / 255 ],
-    'mintcream': [ 245 / 255, 255 / 255, 250 / 255 ],
-    'mistyrose': [ 255 / 255, 228 / 255, 225 / 255 ],
-    'moccasin': [ 255 / 255, 228 / 255, 181 / 255 ],
-    'navajowhite': [ 255 / 255, 222 / 255, 173 / 255 ],
-    'navy': [ 0 / 255, 0 / 255, 128 / 255 ],
-    'oldlace': [ 253 / 255, 245 / 255, 230 / 255 ],
-    'olive': [ 128 / 255, 128 / 255, 0 / 255 ],
-    'olivedrab': [ 107 / 255, 142 / 255, 35 / 255 ],
-    'orange': [ 255 / 255, 165 / 255, 0 / 255 ],
-    'orangered': [ 255 / 255, 69 / 255, 0 / 255 ],
-    'orchid': [ 218 / 255, 112 / 255, 214 / 255 ],
-    'palegoldenrod': [ 238 / 255, 232 / 255, 170 / 255 ],
-    'palegreen': [ 152 / 255, 251 / 255, 152 / 255 ],
-    'paleturquoise': [ 175 / 255, 238 / 255, 238 / 255 ],
-    'palevioletred': [ 219 / 255, 112 / 255, 147 / 255 ],
-    'papayawhip': [ 255 / 255, 239 / 255, 213 / 255 ],
-    'peachpuff': [ 255 / 255, 218 / 255, 185 / 255 ],
-    'peru': [ 205 / 255, 133 / 255, 63 / 255 ],
-    'pink': [ 255 / 255, 192 / 255, 203 / 255 ],
-    'plum': [ 221 / 255, 160 / 255, 221 / 255 ],
-    'powderblue': [ 176 / 255, 224 / 255, 230 / 255 ],
-    'purple': [ 128 / 255, 0 / 255, 128 / 255 ],
-    'red': [ 255 / 255, 0 / 255, 0 / 255 ],
-    'rosybrown': [ 188 / 255, 143 / 255, 143 / 255 ],
-    'royalblue': [ 65 / 255, 105 / 255, 225 / 255 ],
-    'saddlebrown': [ 139 / 255, 69 / 255, 19 / 255 ],
-    'salmon': [ 250 / 255, 128 / 255, 114 / 255 ],
-    'sandybrown': [ 244 / 255, 164 / 255, 96 / 255 ],
-    'seagreen': [ 46 / 255, 139 / 255, 87 / 255 ],
-    'seashell': [ 255 / 255, 245 / 255, 238 / 255 ],
-    'sienna': [ 160 / 255, 82 / 255, 45 / 255 ],
-    'silver': [ 192 / 255, 192 / 255, 192 / 255 ],
-    'skyblue': [ 135 / 255, 206 / 255, 235 / 255 ],
-    'slateblue': [ 106 / 255, 90 / 255, 205 / 255 ],
-    'slategray': [ 112 / 255, 128 / 255, 144 / 255 ],
-    'slategrey': [ 112 / 255, 128 / 255, 144 / 255 ],
-    'snow': [ 255 / 255, 250 / 255, 250 / 255 ],
-    'springgreen': [ 0 / 255, 255 / 255, 127 / 255 ],
-    'steelblue': [ 70 / 255, 130 / 255, 180 / 255 ],
-    'tan': [ 210 / 255, 180 / 255, 140 / 255 ],
-    'teal': [ 0 / 255, 128 / 255, 128 / 255 ],
-    'thistle': [ 216 / 255, 191 / 255, 216 / 255 ],
-    'tomato': [ 255 / 255, 99 / 255, 71 / 255 ],
-    'turquoise': [ 64 / 255, 224 / 255, 208 / 255 ],
-    'violet': [ 238 / 255, 130 / 255, 238 / 255 ],
-    'wheat': [ 245 / 255, 222 / 255, 179 / 255 ],
-    'white': [ 255 / 255, 255 / 255, 255 / 255 ],
-    'whitesmoke': [ 245 / 255, 245 / 255, 245 / 255 ],
-    'yellow': [ 255 / 255, 255 / 255, 0 / 255 ],
-  'yellowgreen': [ 154 / 255, 205 / 255, 50 / 255 ] }
+/**
+ * Converts an CSS color name to RGB color.
+ *
+ * @param   String  s       The CSS color name
+ * @return  Array           The RGB representation, or [0,0,0] default
+ */
+export function css2rgb(s) {
+  var r = 0;
```

Comment:
As debated off line, we both agreed that returning an error (exception) is better than returning a default color.

To throw an exception or to let the program die (color will be undefined) is another debate as the library needs to be consistent. Let's open a new issue to debate with others.

For now, this PR is consistent with the current policy for this function, i.e. let the program die.